src/PAModel/CanvasDocument.cs

```diff
@@ -625,16 +623,19 @@ private int FindMaxEntropyFileName()
         private void RestoreAssetFilePaths()
         {
             // For apps unpacked before this asset rewrite was added, skip the restore step
-            if (_entropy.LocalResourceFileNames == null)
+            if (_entropy?.LocalResourceFileNames == null)
                 return;
 
-            if (_resourcesJson == null || _assetFiles == null)
+            if (_resourcesJson?.Resources == null || _assetFiles == null)
```

Comment:
You would still need to separate it out `if (_resourcesJson== null || _resourcesJson.Resources == null || _assetFiles == null)`